pd/src/info/specific.rs

```diff
@@ -317,8 +318,26 @@ impl SpecificQueryService for Info {
         let rsp = match denom {
             Some(denom) => {
                 tracing::debug!(?id, ?denom, "found denom");
+
+                let base_unit = DenomUnit {
+                    denom: denom.to_string(),
+                    exponent: 0,
+                    aliases: vec![],
+                };
+
                 AssetInfoResponse {
-                    asset: Some(Asset { id, denom }.into()),
+                    denom_metadata: Some(DenomMetadata {
+                        base: base_unit.denom.clone(),
+                        display: base_unit.denom.clone(),
+                        denom_units: vec![base_unit],
+                        penumbra_asset_id: Some(id.into()),
+                        // TODO: Keep track of this metadata and report it for Penumbra's own assets
+                        name: Default::default(),
+                        description: Default::default(),
+                        symbol: Default::default(),
+                        uri: Default::default(),
+                        uri_hash: Default::default(),
+                    }),
```

Comment:
Why are we constructing the `DenomMetadata` manually, rather than just pulling it out of the state?